client/engine/messageservice/p2p-message-service/service.go

```diff
@@ -15,30 +16,44 @@ import (
 	"github.com/libp2p/go-libp2p/core/peer"
 	"github.com/libp2p/go-libp2p/core/peerstore"
 	"github.com/libp2p/go-libp2p/core/protocol"
+	"github.com/libp2p/go-libp2p/p2p/discovery/mdns"
 	"github.com/libp2p/go-libp2p/p2p/transport/tcp"
-	"github.com/multiformats/go-multiaddr"
 	"github.com/statechannels/go-nitro/internal/safesync"
 	"github.com/statechannels/go-nitro/protocols"
 	"github.com/statechannels/go-nitro/types"
 )
 
+// PeerInfo contains information about a peer
+type PeerInfo struct {
+	Id      peer.ID
+	Address types.Address
+
+	// TODO: These are only here for backwards compatibility with the old testground test
+	Port      int
```

Comment:
These fields can be removed once https://github.com/statechannels/go-nitro-testground/pull/161 is merged

Reply:
We can rely on the `PeerStore` , which stores a multiaddress for each of our peers. We don't need to bother storing the port/ipaddress on `PeerInfo`